### PR TITLE
Cleanup NavLink. Part of STCOR-237

### PR DESCRIPTION
**Pull request by mkuklis**
A change required by the refactoring done in: https://github.com/folio-org/stripes-core/pull/389

### CODE REVIEW

File: lib/NavLink/NavLink.js [hunk 1]
```diff
@@ -1,47 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { FormattedMessage } from 'react-intl';
-import NavListItem from '@folio/stripes-components/lib/NavListItem';
 import ServicePointsModal from '../ServicePointsModal';
 
 class NavLink extends React.Component {
```
Comment (MikeTaylor) [inline lib/NavLink/NavLink.js]: Of course this shouldn't really be called NavLink any more.

Comment (mkuklis) [reply lib/NavLink/NavLink.js]: Good catch! I will clean it up.

**Comment by mkuklis** [comment]
@MikeTaylor it turned out we can remove the `NavLink` completely. The `eventHandler` got simplified too. I'm starting to like it.

**Comment by MikeTaylor** [comment]
Wow. This got _simple_!

**Pull request merged**